### ISSUE TITLE
Better feedback about accessing restricted directories and files

### ISSUE THREAD

**Issue by joshas** [MeanEYE/Sunflower#321]
User can open restricted directories (e.g. /root) in file list, but it shows up empty. Would be nice to display message to user that access is restricted, or show emblem on icon, like in Gnome Files.
Currently only error appears in console.

Adding option to elevate privileges and open directory as super-user would be even better.

**Comment by MeanEYE** [comment]
You are right. We should add an emblem, the same way we notify about symlinks.
